app/views/submissions.scala.html

```diff
@@ -24,44 +24,80 @@
 @import forms.Choice
 @import forms.Choice.AllowedChoiceValues.Submissions
 @import views.Title
+@import views.html.components.gds._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.BackButton
 
-@this(main_template: views.html.main_template)
+@this(
+        govukLayout: gdsMainTemplate,
+        pageTitle: pageTitle,
+        govukTable : GovukTable,
+        linkButton: linkButton
+)
 
 @(submissions: Seq[(Submission, Seq[Notification])])(implicit request: Request[_], messages: Messages)
 
-@main_template(
-    title = Title("submissions.title"),
-    fullWidth = true
-) {
+@headCell(key: String) = @{
+    HeadCell( content = Text(messages(s"submissions.$key.header")) )
+}
 
-    @components.back_link(
-        href = routes.ChoiceController.displayPage(Some(Choice(Submissions))),
-        backMessage = "site.backToSelectionPage"
+@ucrLink(submission: Submission) = @{
```

Comment:
You can use the components.gds.link component to do this.  Could even add some accessible "hidden" text using this method (even though the prototype doesn't have this - could check with Chris/James)